site/cypress/e2e/polls.spec.js

```diff
@@ -116,8 +116,34 @@ describe('Test cases revolving around polls functionality', () => {
         cy.get('#chartContainer').contains('2 * 3 = 6');
         cy.get('#chartContainer').contains('8 / 2 = 3');
         cy.get('#chartContainer').contains('1 * 3 = 4');
-    });
+        cy.go('back');
 
+        //tomorrow's poll test, name poll 5
+        cy.contains('New Poll').click();
+        cy.get('#poll-name').type('Poll 5');
+        cy.get('#poll-question').type('What is your favorite class?');
+        cy.get('#poll-date').clear({force: true});
+        const today = new Date(Date.now());
+        const tomorrow = new Date(today);
+        tomorrow.setDate(tomorrow.getDate() + 1); // set date to one day forward
+        cy.get('#poll-date').type(tomorrow.toISOString().substring(0, 10), {force: true});
+        cy.contains('Add Response').click();
+        cy.get('#response_0_wrapper').children(':nth-child(3)').check();
+        cy.get('#response_0_wrapper').children(':nth-child(4)').type('Data Structures');
+        cy.get('h1').click();
+
+        // submit and verify on main polls page, poll should be in tomorrow table
+        cy.get('#poll-form-submit').click();
+        cy.url().should('include', 'sample/polls');
+        cy.contains('Poll 5');
+        cy.get('#tomorrow-table').contains('Poll 5');
+
+        // delete the poll
+        cy.contains('Poll 5').siblings(':nth-child(2)').click();
```

Comment:
should I move all tomorrow's poll test cases to the bottom section where it says "should verify making, editing, and deleting poll works as expected"

Reply:
> In general, each test case should clean up after itself. Each test case should delete any polls it creates.
> 
> That said, I think the broader issue here is that these changes are placed in the wrong spot. This test case is specifically testing the sample polls which are generated by the sample course generation script. I believe that all of these changes can be moved to a completely new test case since they seem to be completely independent of everything else.

should I move all tomorrow's poll test cases to the bottom section where it says "should verify making, editing, and deleting poll works as expected"

Reply:
All of the test code you wrote here is completely independent of everything else in the test case you added it to.  Given that it's entirely independent (creating, modifying, and deleting a poll for testing purposes), this code should be moved to a new test case.